lessons/git/lesson.md

```diff
@@ -36,10 +36,11 @@ together:
 
 * http://codeasmanuscript.github.io/workshops/lessons/git/cheatsheet/
 
-Commands:
+Then, fork @mbonsma's repository into your own GitHub account. After
+wards do these commands:
 
     cd ~/Desktop
-    git clone https://github.com/lwjohnst86/studyGroup.git
+    git clone https://github.com/your-accout-name/studyGroup.git
```

Comment:
Well, emacs, but I use vim keybindings + `:` commands (since Emacs 
bindings are ... terrible...). I just didn't register that is what you 
were referring too haha

On 15-06-17 06:52 PM, Elliott Sales de Andrade wrote:
>
> In lessons/git/lesson.md 
> <https://github.com/mbonsma/studyGroup/pull/9#discussion_r32684081>:
>
> >
> >      cd ~/Desktop
> > -    git clone https://github.com/lwjohnst86/studyGroup.git
> > +    git clone https://github.com/your-accout-name/studyGroup.git
>
> Not a |vim| user, eh?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mbonsma/studyGroup/pull/9/files#r32684081>.
>